hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/dstable/DistributionDetails.java

```diff
@@ -113,18 +113,17 @@ protected String getEditButtonId() {
     }
 
     @Override
-    protected Boolean onLoadIsTableRowSelected() {
-        return !(managementUIState.getSelectedDsIdName().isPresent()
-                && managementUIState.getSelectedDsIdName().get().isEmpty());
+    protected boolean onLoadIsTableRowSelected() {
+        return managementUIState.getSelectedDsIdName().map(selected -> !selected.isEmpty()).orElse(false);
```

Comment:
I think this should be
`managementUIState.getSelectedDsIdName().map(selected -> selected.isEmpty()).orElse(true);`

Please double check